rpcserver.go

```diff
@@ -2049,9 +2049,31 @@ func (r *rpcServer) FundVirtualPsbt(ctx context.Context,
 
 	case req.GetRaw() != nil:
 		raw := req.GetRaw()
+		prevIDs := []asset.PrevID{}
```

Comment:
nit: `var prevIDs []asset.PrevID` style is preferred over empty slice initialization.